src/CMakeLists.txt

```diff
@@ -732,12 +768,19 @@ if(${WITH_RBD})
   set(DENCODER_EXTRALIBS
       ${DENCODER_EXTRALIBS}
       rbd_replay_types)
+  set(dencoder_srcs
+      ${dencoder_srcs}
+      $<TARGET_OBJECTS:krbd_objs> )
 endif(${WITH_RBD})
+if(${WITH_KVS})
+  set(dencoder_srcs
```

Comment:
cmake expect `CMAKE_*_FLAGS` to be a single string instead of lists. guess they are exceptions.